chain/messagepool/messagepool.go

```diff
@@ -240,10 +240,13 @@ func (ms *msgSet) add(m *types.SignedMessage, mp *MessagePool, strict, untrusted
 			// check if RBF passes
 			minPrice := ComputeMinRBF(exms.Message.GasPremium)
 			if types.BigCmp(m.Message.GasPremium, minPrice) >= 0 {
-				log.Infow("add with RBF", "oldpremium", exms.Message.GasPremium,
+				log.Debugw("add with RBF", "oldpremium", exms.Message.GasPremium,
 					"newpremium", m.Message.GasPremium, "addr", m.Message.From, "nonce", m.Message.Nonce)
 			} else {
-				log.Info("add with duplicate nonce")
+				log.Debugf("add with duplicate nonce. message from %s with nonce %d already in mpool,"+
+					" increase GasPremium to %s from %s to trigger replace by fee: %w",
```

Comment:
this `%w` formatting directive is invalid in the log, it's only valid in the error wrapping function; it should be `%s`.

Reply:
hi @vyzo thank you

Reply:
FYI to all, the best format to use for errors is `%v`, or if you want detailed info `%+v`.

Reply:
it's not necessary here - it's not a structured error.